c7n/resources/rds.py

```diff
@@ -77,6 +78,7 @@
 filters.register('marked-for-op', tags.TagActionFilter)
 filters.register('health-event', HealthEventFilter)
 actions.register('auto-tag-user', AutoTagUser)
+actions.register('dummy', DummyAction)
```

Comment:
umm.. why the dummy action, this should be localized to tests if that's the intent.